package.json

```diff
@@ -19,9 +19,6 @@
     "prebuild": "rimraf dist",
     "build": "nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
-    "start": "nest start",
-    "start:dev": "nest start --watch",
-    "start:debug": "nest start --debug --watch",
```

Comment:
These didn't seem useable.  I think I could also remove "start:prod" and maybe even the "test" and jest config as well since the package doesn't seem to be using any of that